src/link/styles.scss

```diff
@@ -54,6 +54,11 @@
       @include styles.link-font-size-style(map.get(constants.$link-font-sizes, $font-size));
     }
   }
+
+  @include styles.with-direction('rtl') {
+    align-items: center;
+    display: inline-flex;
+  }
```

Comment:
Why is this rtl only? I do not see any direction related behavior in these lines

Reply:
Ok to ship this, but let's do a follow up because it is clearly a debt

Reply:
Agreed!